README.md

```diff
@@ -135,8 +135,47 @@ See [rules_haskell_examples][] for examples of using these rules.
 
 [rules_haskell_examples]: https://github.com/tweag/rules_haskell_examples
 
+
 ## For `rules_haskell` developers
 
+### Reference a local checkout of `rules_haskell`
+
+When you develop on `rules_haskell`, you usually do it in the context
+of a different project that has `rules_haskell` as a `WORKSPACE`
+dependency, like so:
+
+```
+http_archive(
+    name = "io_tweag_rules_haskell",
+    strip_prefix = "rules_haskell-" + version,
+    sha256 = …,
+    urls = …,
+)
+```
+
+To reference a local checkout instead, there’s two methods:
+
+1) Use the [`--override_repository`][override_repository]
+   command line option:
+   
+   ```
+   bazel build/test/run/sync \
+     --override_repository io_tweag_rules_haskell=/path/to/checkout
+   ```
+   
+2) Edit your `WORKSPACE` to use [`local_repository`][local_repository]
```

Comment:
Remove this option. This is the wrong way to do it.

Reply:
But bazelrc is even more right. It's the only mechanism that allows you to override on a per user basis without fearing committing the dirty changes in the WORKSPACE by mistake. See https://docs.bazel.build/versions/master/best-practices.html#bazelrc.

I'm not a fan of overlong README's. The bar should be high as to what content goes there. Let's avoid repeating the documentation there for things are bad practices anyways.

Reply:
Okay, replaced it with “If you don’t want to type that every time, temporarily add it to
   if you want a more permanent setup (just don’t commit it):	`.bazelrc`.”

Reply:
Add the link I pointed you to.

Reply:
Oh, right. Done.